### DISCUSSION TITLE
Cannot get YouTube videos added to Wiki

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#6009] [Error / Bug Report]
Page:
http://wiki.ccnstudios.com/en/freeform/characters/humanoid/unity-chan 

I added the link, but cannot Embed:

<iframe width="560" height="315" src="https://www.youtube.com/embed/EMm7zmT_8W0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

I use the insert media then enter the url and nothing happens, the final rendering has no media!

Apparantly  there is an option to turn on iframe:

https://prnt.sc/vo3f08  

but I cannot find it on my Rendering menu item

**Comment by NGPixel** [comment]
It's under Admin > Security.

**Reply by NGPixel** [reply]
You also need to disable HTML Sanitization under Admin > Rendering > HTML > Security and enable iframes from there.

**Comment by darashayda** [comment]
I did what you instructed and no Youtube

http://wiki.ccnstudios.com/en/freeform/characters/humanoid/unity-chan

**Comment by darashayda** [comment]
Got it to work, I needed to be in MarkDown Editor. ... thanx all is good

**Reply by Xorbite** [reply]
Hi, how did you get iframes to work in the MarkDown editor if you don't mind me asking?

**Comment by darashayda** [comment]
Of I do not mind, my honour to be of any use. 

See the screenshots 

1. Most media vendors e.g. Youtube or Vimeo or ithch.io  issue you the embed code listing, which you simply copy and paste as text into the MarkDown Editor, where ever line of code you like to paste2. 2. 

In the case of this particular page of ours, source code obtained from Youtube, the process for acquiring it is in the attached images. 

<iframe width="560" height="315" src="https://www.youtube.com/embed/EMm7zmT_8W0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

If the vendor did not offer the iframe code, the syntax  and options more or less are similar to the above, which are a part of the HTML standard. 

I attache images, in case they do not show I will repost. 

Let me know if this is adequate 

D


<img width="988" alt="Screen Shot 2023-02-20 at 4 20 10 PM" src="https://user-images.githubusercontent.com/5667455/220157546-31a9c840-6aa7-4180-a0cf-761443a1aa84.png">
<img width="988" alt="Screen Shot 2023-02-20 at 4 20 23 PM" src="https://user-images.githubusercontent.com/5667455/220157598-f9fd9885-63b9-4800-9b71-28ae686a8fb3.png">
<img width="1206" alt="Screen Shot 2023-02-20 at 4 18 48 PM" src="https://user-images.githubusercontent.com/5667455/220157609-0b556911-dc92-4a03-8f72-c75e32bae738.png">

**Reply by Xorbite** [reply]
Cheers mate, it indeed worked perfectly

**Reply by darashayda** [reply]
Terrific! Hit me if you ever need help again. We have to contribute to the community. 
The product is high quality but the docs are a bit confusing for my discerning tase :)

Dara

**Reply by Xorbite** [reply]
Agreed and you're totally right, the documentation is missing a lot of crucial parts

**Comment by darashayda** [comment]
As NGPIXEL advised you to go to your admin dasboard:

**Disable HTML Sanitization under Admin > Rendering > HTML > Security and enable iframes from there.** 

For me that was the missing link! 

If you have issues with that, let me know 

Dara

**Comment by fumbles** [comment]
https://github.com/requarks/wiki/discussions/4580#discussioncomment-5327035

I was able to keep HTML Sanitization with using this script and just pasting the youtube link into the page.
